{"event": "verbose", "uuid": "d75dd792-9ff8-40b1-9edc-af1b713c617c", "counter": 6, "stdout": "\u001b[0;31mThe error appears to be in '/home/duchess/Documents/src/github.com/edwardtheharris/ansible-kcp/tmp_dir_sphinx_ansible/temp_file.yaml': line 1, column 3, but may\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "2e285528-ded4-4434-87f7-561bcdb4141d", "created": "2024-07-23T18:49:25.517435+00:00"}